core/src/main/java/com/airbnb/aerosolve/core/transforms/StringCrossFloatTransform.java

```diff
@@ -1,23 +1,37 @@
 package com.airbnb.aerosolve.core.transforms;
 
 import com.airbnb.aerosolve.core.FeatureVector;
+import com.airbnb.aerosolve.core.features.Features;
 import com.airbnb.aerosolve.core.util.Util;
-
 import com.typesafe.config.Config;
-import java.util.HashMap;
-import java.util.Set;
-import java.util.Map;
 
+import java.util.*;
+
+/*
+  if no prefix/keys provided, then cross whole family.
+  prefix used in string, and keys used in float.
+  to cross RAW features, use Features.RAW
```

Comment:
please add more details about what is key, what is prefix, without actually looking at the code, it's not clear that prefix is for string family and key is for float feature. and it's also not clear what exactly is prefix, people may think it means the prefix of a feature name, but it's actually feature name.